projects/dynamicforms/src/core/services/file/file.service.ts

```diff
@@ -78,4 +79,13 @@ export class FileService {
       });
     }
   }
+
+  getDocument(document, event) {
+    event.preventDefault();
```

Comment:
This shouldn't be in service method. It should be specific to component. Maybe someone may not attach this to `<a>` tag and doesn't want to prevent default browser behavior.